### DISCUSSION TITLE
`Feature not available for users using GPG`

### DISCUSSION THREAD

**Discussion by scallaway** [jesseduffield/lazygit#2403] [General]
Hi there,

What's the reason for not allowing quick renaming of commits if you're using GPG signing?

At my company, it's a requirement to sign all commit messages, but I also really like the very quick renaming that LazyGit has to offer.

I'd love to better understand this, as maybe I could offer some help to combat it.

Thanks

**Comment by pprotas** [comment]
Any updates on this?

**Comment by corytheboyd-smartsheet** [comment]
I sign all of my commits, and was able to rename commits in lazygit after adding this to the lazygit configuration:

```yaml
git:
    overrideGpg: true
```

However I don't recall if there was some other step(s) involved... but I can confirm that my commits are signed and my commit message renames work :smile:

**Reply by 0-sv** [reply]
It also showed me that this config option exists when I upgraded `lazygit` to `0.57.0` in the "breaking changes info" modal. Works like a charm, thanks!
